R/pruneScores.R

```diff
@@ -2,9 +2,9 @@
 #'
 #' Remove low-quality assignments based on the cell-label score matrix returned by \code{\link{classifySingleR}}.
 #'
-#' @param scores SingleR output, or a Matrix of scores per cell (row) and label (column) generated during assignment by \code{\link{classifySingleR}}.
+#' @param results SingleR output generated during assignment by \code{\link{classifySingleR}}.
```

Comment:
`A \linkS4class{DataFrame} containing the output generated by \code{\link{SingleR}} or \code{\link{classifySingler}}`.

Reply:
changed to this